GG/src/DropDownList.cpp

```diff
@@ -53,13 +58,40 @@ class ModalListPicker : public Wnd
 
     mutable SelChangedSignalType SelChangedSignal; ///< the selection change signal object for this DropDownList
 
+    DropDownList::iterator CurrentItem();
+
+    /** Select \p it  in m_lb_wnd is not none and return the new selected or none if no change.*/
```

Comment:
It looks like I left out the word "if" as well as phrased is awkwardly.  I've rephrased it.